### ISSUE TITLE
No TypeScript types for @emotion/native

### ISSUE THREAD

**Issue by ngirardin** [emotion-js/emotion#1851]
**Current behavior:**

Hi guys, I'm using Typescript and it seems that VSCode can't find the types for @emotion/native.

I've tested on macOS and Windows, looked in closed issues and on the web but it seems that I'm the only having this issue.

**To reproduce:**

```
npx react-native init ReactNativeEmotion --template react-native-template-typescript
yarn add @emotion/core @emotion/native 
```

<img width="865" alt="Screenshot 2020-04-24 at 09 32 24" src="https://user-images.githubusercontent.com/579776/80186614-8e1ab300-860e-11ea-8cb5-c9c4a157e3ed.png">

**Expected behavior:**

VSCode display the types like with the @emotion/core package.

**Environment information:**

```
    "@emotion/core": "^10.0.28",
    "@emotion/native": "^10.0.27",
    "react": "16.11.0",
    "react-native": "0.62.2"
```

Thanks for your help :)

**Comment by TheRohitSharma** [comment]
@ngirardin Same issue, but I believe it's because there aren't any types: https://github.com/emotion-js/emotion/tree/master/packages/native

**Comment by philscott-dev** [comment]
They're available via ```@emotion/native@^11.0.0-next.12```. Hopefully v11 gets released soon.

**Comment by Andarist** [comment]
What @philscott-dev said. The last big change is about to land in v11 - https://github.com/emotion-js/emotion/pull/1817 . After that, I will have to go through all issues and PRs but I don't expect many new things popping up to do before the final release - there still might be some minor stuff to do. You can track progress here: https://github.com/emotion-js/emotion/milestone/1

Going to close this as we don't plan to backport those typings to v10. I hope you understand.
